index.js

```diff
@@ -122,31 +124,73 @@ module.exports = class Config {
       }
       filePath = path.resolve(relativePath)
     }
+    else if (options.path) {
+      filePath = path.resolve(options.path)
+    }
+    else {
+      throw new Error('No file path was specified')
+    }
 
     const fileExtension = path
       .extname(filePath)
       .slice(1)
-    const fileContent = fs.readFileSync(filePath)
-    let configObject
+    let fileContent
```

Comment:
Assign `null`, so when you debug you can distinguish a typo in a variable and a not properly initialized variable.

Reply:
What should be different if it's null? It throws an error if path isn't set either way...

Reply:
It either throws with `undefinied` if the variable is not definied or `null`, if you just forgot to initialize the variable correctly.

Reply:
It throws `new Error('No file path was specified')` no matter of filePath is `undefined` or `null`

Reply:
It is a matter of writing code, so it is easy to debug. It is not about the specific case. Imagine you refactor the code and make a mistake. In that case it would be nice to have the distinction. Anyway, it is a matter of the in JS I guess, so I can live with it.

Reply:
Ah ok. Now I understand what you mean. Next time ;-)